src/vscode-bicep/src/panes/deploy/view.ts

```diff
@@ -247,6 +247,9 @@ export class DeployPaneView extends Disposable {
     const scriptUri = this.webviewPanel.webview.asWebviewUri(
       vscode.Uri.joinPath(this.extensionUri, "out", "deployPane.js"),
     );
+    const codiconCssUri = this.webviewPanel.webview.asWebviewUri(
```

Comment:
I reintroduced this because the new library already has a codicon element available (`VscodeIcon`), and I was running into some super cryptic React errors trying to use the element from the vscode-bicep-ui project (https://react.dev/warnings/invalid-hook-call-warning).